process.go

```diff
@@ -109,18 +110,38 @@ func (proc *concurrentProcess) wait() {
 // newCommandRunner creates new external command runner for given executable. The executable path
 // is resolved in this function.
 func (proc *concurrentProcess) newCommandRunner(exe string, combineOutput bool) (*externalCommand, error) {
-	p, err := execabs.LookPath(exe)
+	var args []string
+	p, args, err := findExe(exe)
 	if err != nil {
 		return nil, err
 	}
 	cmd := &externalCommand{
 		proc:          proc,
 		exe:           p,
+		args:          args,
 		combineOutput: combineOutput,
 	}
 	return cmd, nil
 }
 
+func findExe(exe string) (string, []string, error) {
+	p, err := execabs.LookPath(exe)
+	if err == nil {
+		return p, nil, nil
+	}
+	// See if the command string contains args. As it is best effort, we do not
+	// handle parse errors.
+	if exeArgs, _ := shellwords.Parse(exe); len(exeArgs) > 0 {
+		// We want to return the original error if this command isn't found so
+		// do not overwrite it.
+		if p, err2 := execabs.LookPath(exeArgs[0]); err2 == nil {
```

Comment:
```suggestion
		if p, err := execabs.LookPath(exeArgs[0]); err == nil {
```

Would you shadow the outer `err` variable? It is okay because this variable is very short-lived.